modules/osdk-ansible-k8s-module-testing-locally.adoc

```diff
@@ -10,10 +10,16 @@ local machine as opposed to running and rebuilding the Operator each time.
 
 .Procedure
 
+. Install the `community.kubernetes` collection:
+
```

Comment:
This needs a "+" on this line so that the following code block stays with step 1. It's also causing the numbering for the next step to restart at 1.